cluster/gce/config-test.sh

```diff
@@ -156,3 +156,10 @@ HAIRPIN_MODE="${HAIRPIN_MODE:-promiscuous-bridge}" # promiscuous-bridge, hairpin
 
 # Optional: if set to true, kube-up will configure the cluster to run e2e tests.
 E2E_STORAGE_TEST_ENVIRONMENT=${KUBE_E2E_STORAGE_TEST_ENVIRONMENT:-false}
+
+# A list of images to pre-pull for the e2e, #25277
+function get_image_list {
+  grep -Iiroh "gcr.io/google_.*" "${KUBE_ROOT}/test/e2e" | sed -e "s/[,\")}]//g" | awk '{print $1}' | sort | uniq | tr '\n' ' '
```

Comment:
The generated list doesn't include critical daemonset images, such as kube-proxy, fluentd, etc. We do have flakiness introduced by those images too. 

I know at this stage, cluster is ready. When cluster is ready, those daemonsets have to be at ready stage. I am thinking prepopulate those images before cluster ready stage, which would be great.